sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -262,19 +264,36 @@ public static Sink sink(Schema schema) {
     @Nullable
     abstract String getJsonSchema();
 
+    @Nullable
```

Comment:
You can remove this `@Nullable` by setting the `CompressionCodecName.UNCOMPRESSED` when you build the object `write, sink`. This way you avoid also the `if == null` comparison .

Reply:
ok